addons/udes_hierarchical_tree_view/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "UDES Hierarchical Tree View",
+    "summary": """
+        UDES Hierarchical Tree View
+    """,
+    "description": """
+         UDES Hierarchical Tree View
+    """,
+    "author": "Unipart digital",
+    "website": "http://github/unipartdigital/udes-open",
+    "category": "Tools",
+    "version": "0.1",
+    "depends": ["base", "web", "mail", "product"],
```

Comment:
Q: Is this `product` dependency temporary? (Considering your comment in `product_views.xml`.

Also a bit unsure if the `mail` dependency is needed?

Reply:
product is temporary, and you're right looks like mail is not necessary as a dep